### PR TITLE
Theming docs update

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Update theming docs in storybook since it is quite outdated since FluentUI and FluentUI N* icons have been merged so FluentThemeProvider is no longer mandatory. Also, we call it ACS UI Library instead of ACS UI SDK.

# Why

Better documentation

# How Tested

Local deploy of storybook

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by alcail — APPROVED** [review]

File: packages/storybook/stories/Theming.stories.mdx [hunk 1]
```diff
@@ -1,60 +1,50 @@
...
-`FluentThemeProvider` does two things here:
-
-1. `FluentThemeProvider` loads the icons used by components such as `CameraButton`
-2. `FluentThemeProvider` applies theme (light theme by default) to the `ControlBar`
-
-To illustrate how another theme can be applied, import `darkTheme` from `@azure/communication-react`
-and assign it to the `fluentTheme` property.
+To theme this ControlBar, import `FluentThemeProvider` from `@azure/communication-react` and wrap it around the `ControlBar`.
+Then import `darkTheme` from `@azure/communication-react` and assign it to the `fluentTheme` property of `FluentThemeProvider`.
+The import `darkTheme` is a [Theme](https://aka.ms/fluentui#/controls/web/references/theme) object from
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Theming.stories.mdx:30]: nit: Delete "The import"

File: packages/storybook/stories/Theming.stories.mdx [hunk 2]
```diff
@@ -1,60 +1,50 @@
...
-
 ## FluentThemeProvider Props
 
 <Props of={FluentThemeProvider} />
 
 ## Creating your own theme
 
-If you wish to use your own theme, we strongly advise using Microsoft's [Fluent UI Theme Designer](https://aka.ms/themedesigner).
-Please ensure that the Accessibility checker on the same page passes when creating your own theme.
+The imports `lightTheme` and `darkTheme` are available from `@azure/communication-react`. But if you wish to use your own theme,
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Theming.stories.mdx:45]: nit: Delete "The imports"

File: packages/storybook/stories/Theming.stories.mdx [hunk 3]
```diff
@@ -1,60 +1,50 @@
...
+we strongly advise using Microsoft's [Fluent UI Theme Designer](https://aka.ms/themedesigner) to create your theme. Please ensure
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Theming.stories.mdx:46]: nit: strongly advise -> recommend

I think it's worth being polite in these docs.

File: packages/storybook/stories/Theming.stories.mdx [hunk 4]
```diff
@@ -91,16 +81,18 @@ export const lightTheme = {
...
+use the component's `styles` prop like in the example below.
 
 <Source code={ThemedButtonText} />
 
 <Canvas withSource="none">
   <ThemedButton />
 </Canvas>
 
-Should you override your own mappings, please note that ACS SDK UI components abide by the general Fluent UI
+Should you override your own mappings, please note that ACS UI Library components abide by the general Fluent UI
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Theming.stories.mdx:95]: "Override your own mappings" sounds wrong.

**Review by prprabhu-ms — APPROVED** [review]

**Pull request merged**